docs/data-imports.md

```diff
@@ -0,0 +1,52 @@
+# Data Imports
+
+The data imports pick up where the ETL leaves off by copying and importing the
+CSV files into the Postgresql database. This document describes that process in
+more detail and also some of the schema choices that were made to optimize the
+imports for speed.
+
+The examples below will use Firefox on Android as the example, as it is a Glean
+based product and will be the example for future products in GLAM going forward.
+
+## Schema
+
+The Firefox for Desktop (aka fenix) product has 2 primary tables assocated with
+its data. The aggregation table and the materialized view on top of that data.
+
+The aggregation table exist so imports can be as fast as possible. It has
+minimal indexes and constraints so data can import quickly. The only index is
+the primary key. The only constraint is a unique constraint comprised of the
+dimensions of the data (e.g. channel, build ID, metric, etc.) so that we can
+assure we have only 1 row across these dimensions. This unique constraint is
+also used in the "upsert" so that if new data comes in we insert it, or if a
+matching record exists we update the data (counts, histograms, and percentiles).
+
+The materialized view exists so the API queries are as fast as possible. It is a
+direct copy of the aggregation table but with added indexes, roughly to match
+the top level selectors in GLAM so the SQL `WHERE` clauses are fast. Since the
```

Comment:
> It is a direct copy of the aggregation table but with added indexes, roughly to match the top level selectors in GLAM so the SQL `WHERE` clauses are fast.

Where in the code is this part `it is a direct copy of the aggregation table but with added indexes`?

Reply:
It is in the migrations that created the tables. I could add a link to it here, but if those files ever change this would need to be updated.